### PR TITLE
feat: monorepo implemented

### PR DESCRIPTION
**Pull request by ZarcoNontol**
Added monorepo so we can test esbuild-server as external package instead of re-publishing it over and over.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+language: node_js
```
Comment (rzadp) [inline .travis.yml:1]: Do we use travis?

File: packages/esbuild-book-knobs/dist/src/components/Knobs.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export declare const Knobs: any;
```
Comment (rzadp) [inline packages/esbuild-book-knobs/dist/src/components/Knobs.d.ts:1]: Commiting dist?

File: packages/esbuild-server/.gitignore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+node_modules
+dist
+.DS_Store
+.idea
```
Comment (rzadp) [inline packages/esbuild-server/.gitignore:4]: move gitignore to root maybe?

File: packages/yarn.lock [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```
Comment (rzadp) [inline packages/yarn.lock:2]: No longer needed

File: common/config/rush/version-policies.json [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+ * More documentation is available on the Rush website: https://rushjs.io
+ */
+
+/**
+ * A list of version policy definitions.  A "version policy" is a custom package versioning
+ * strategy that affects "rush change", "rush version", and "rush publish".  The strategy applies
+ * to a set of projects that are specified using the "versionPolicyName" field in rush.json.
+ */
+[
+  // {
```
Comment (rzadp) [inline common/config/rush/version-policies.json:12]: I think we need a version policy.

Comment (ZarcoNontol) [reply common/config/rush/version-policies.json:12]: I will add it in a separate PR :)

**Pull request merged**